docs/source/asr/speaker_diarization/datasets.rst

```diff
@@ -1,72 +1,134 @@
+	
 .. code-block:: bash
 
-  find $PWD/{data_dir} -iname '*.wav' > path_to_audiofiles.txt
+  {"audio_filepath": "/path/to/audio_file", "offset": 0, "duration": null, label: "infer", "text": "-", "num_speakers": null, "rttm_filepath": "/path/to/rttm/file", "uem_filepath": "/path/to/uem/file"}
 
-.. note::
-  We expect audio and the corresponding RTTM files to have the same base name and the name should be unique (uniq-id).
+We refer to this file as manifest file and the path of the manifest file should be provided in Hydra configuration as follows:
 
-oracle_num_speakers:
-  - if number of speakers is known input it or pass null if not known. Accepts int or path to file containing uniq-id with num of speakers of that session 
+.. code-block:: bash
+   
+	diarizer.manifest_filepath="path/to/manifest/input_manifest.json"
 
-  Sample line (uniq-id `iaaa` session has 2 unique speakers )::
-    
-    iaaa 2
+In the input manifest file, audio_filepath item is required.
+
+
+.. note::
+	We expect all the provided files (e.g. audio, rttm, text) to have the same base name and the name should be unique (uniq-id).
 
-path2groundtruth_rttm_files [Optional]:
-  - To evaluate the diarizer system with known rttm files, One needs to provide a txt file (path2groundtruth_rttm_files) for groundtruth label files.
-  use above mentioned ``find`` command to get all reference rttm files (use '\*.rttm' as search pattern)
+``audio_filepath`` (Required):
+  
+  a file containing absolute paths to audio files or list of paths to audio files. 
 
-Each groundtruth label file should be in NIST Rich Transcription Time Marked (RTTM) format. Take one line from a RTTM file for example:
+``num_speakers`` (Optional):
+    If the number of speakers is known, provide the integer number or assign null if not known. 
+	
+``rttm_filepath`` (Optional):
+  
+    To evaluate a diarization system with known rttm files, one needs to provide Rich Transcription Time Marked (RTTM) files as ground truth label files. If RTTM files are provided, the diarization evaluation will be initiated.
+	Here is one line from a RTTM file as an example:
 
 .. code-block:: bash
 
   SPEAKER TS3012d.Mix-Headset 1 331.573 0.671 <NA> <NA> MTD046ID <NA> <NA>
 
+``text`` (Optional):
+    Ground truth transcription for diarization with ASR inference. Provide the ground truth transcription of the given audio file in string format
+
+.. code-block:: bash
+
+  {"text": "this is an example transcript"}
 
-oracle_vad_manifest [Optional but suggested for good performace. This reduces FPR and TPR]:
-  - To perform just oracle diarization, that is taking speech activity time stamps from groundtruth RTTMs or another VAD from wild instead of NeMo VAD output, ``diarizer`` expects an oracle manifest json file that contains paths to audio files with offset for start time and duration of segment.
+``uem_filepath`` (Optional):
```

Comment:
Here as well Optional keyword is boldly formatted. As well as the next line. Check other similar cases as well.

Reply:
This is fixed by controlling the very first space